test/format/config.c

```diff
@@ -358,6 +375,25 @@ config_run(void)
 {
     config_random(tables[0], false); /* Configure the remaining global name space. */
 
+    /*
+     * Limit the number of tables to 5 if realloc exact and realloc malloc are both on and not all
+     * explicitly set to reduce the running time to acceptable level.
+     */
+    if ((!config_explicit(NULL, "debug.realloc_exact") ||
+          !config_explicit(NULL, "debug.realloc_malloc")) &&
+      GV(DEBUG_REALLOC_EXACT) && GV(DEBUG_REALLOC_MALLOC) && ntables > 5) {
+        ntables = 5;
```

Comment:
Make all of these '5' into `REALLOC_MAX_TABLES`.

But the line above is setting a global variable to 5 down from a larger number. Currently that global variable is set in one place only, now this will make it two. That one place is the end of `config_table_extend`. In that location we've allocated memory and array entries. Do we have to worry about any memory leaks or anything?

Reply:
I don't worry. We allocate more memory for the array but it is small.